include/crm/common/resources.h

```diff
@@ -95,14 +95,17 @@ enum pe_find {
     //! Also match anonymous clone instances by base name
     pcmk_rsc_match_anon_basename    = (1 << 1),
 
+    //! Match only clone instances, by clone or instance ID
+    pcmk_rsc_match_clone_only       = (1 << 2),
```

Comment:
> only clone instances

Looking at `native_find_rsc()`, I think this could also match for `rsc` as the clone's collective resource, if you think that matters semantically.
```
    if (flags & pe_find_clone) {
        const char *rid = ID(rsc->xml);

        if (!pe_rsc_is_clone(pe__const_top_resource(rsc, false))) {
            match = false;
    
        } else if (!strcmp(id, rsc->id) || pcmk__str_eq(id, rid, pcmk__str_none)) {
            match = true;
        }
```

Reply:
Good point